general/clients/index.md

```diff
@@ -281,6 +292,17 @@ A third party app with direct play and HDR support. Available on iOS and Apple T
 
 - [Website](https://mrmc.tv)
 
+### Jellyfin Audio Player
+
+A third party standalone music streaming app. Includes full support for background audio and casting.
+
+**Status** ✅ In Development, 3rd-Party
+
+**Links:**
+
+- [GitHub](https://github.com/leinelissen/jellyfin-audio-player)
+- [TestFlight](https://testflight.apple.com/join/cf2AMDpx)
+
```

Comment:
Even if your client supports multiple platforms, you probably shouldn't copy the information in more than one place. The multiplatform section below might work, or adding a small section in the description with available platforms.

Reply:
Fixed.